### PR TITLE
introspector for persistent executor

### PR DESCRIPTION
**Pull request by njr-11**
Create an introspector for persistent executor, then write an initial test that captures a dump of the server and validates some of the output that it generates.

### CODE REVIEW

File: dev/com.ibm.ws.concurrent.persistent/src/com/ibm/ws/concurrent/persistent/internal/PersistentExecutorIntrospector.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+/**
+ * Introspector for persistent executors.
+ */
+@Component(configurationPolicy = ConfigurationPolicy.IGNORE)
+public class PersistentExecutorIntrospector implements Introspector {
+    @Override
+    @Trivial
+    public String getIntrospectorName() {
+        System.out.println("getIntrospectorName");
```
Comment (nmittles) [inline dev/com.ibm.ws.concurrent.persistent/src/com/ibm/ws/concurrent/persistent/internal/PersistentExecutorIntrospector.java]: Did you intend to leave System.out.println statements?

File: dev/com.ibm.ws.concurrent.persistent/src/com/ibm/ws/concurrent/persistent/internal/PersistentExecutorIntrospector.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        return "Persistent timers/tasks diagnostics";
+    }
+
+    @Override
+    public void introspect(PrintWriter out) throws Exception {
+        SecureAction priv = AccessController.doPrivileged(SecureAction.get());
+        BundleContext bundleContext = priv.getBundleContext(FrameworkUtil.getBundle(getClass()));
+
+        for (ServiceReference<PersistentExecutor> ref : priv.getServiceReferences(bundleContext, PersistentExecutor.class, "(!(com.ibm.wsspi.resource.ResourceFactory=true))")) {
+            System.out.println("Found service reference: " + ref);
```
Comment (nmittles) [inline dev/com.ibm.ws.concurrent.persistent/src/com/ibm/ws/concurrent/persistent/internal/PersistentExecutorIntrospector.java]: Same question as above.

File: dev/com.ibm.ws.concurrent.persistent/src/com/ibm/ws/concurrent/persistent/internal/PersistentExecutorIntrospector.java (hunk 1 above)
Comment (njr-11) [reply dev/com.ibm.ws.concurrent.persistent/src/com/ibm/ws/concurrent/persistent/internal/PersistentExecutorIntrospector.java]: good catch, those were just for debugging locally. I'll remove them.

File: dev/com.ibm.ws.concurrent.persistent/src/com/ibm/ws/concurrent/persistent/internal/PersistentExecutorIntrospector.java (hunk 2 above)
Comment (njr-11) [reply dev/com.ibm.ws.concurrent.persistent/src/com/ibm/ws/concurrent/persistent/internal/PersistentExecutorIntrospector.java]: good catch, those were just for debugging locally. I'll remove them.

**Comment by njr-11** [comment]
#build
#run-libby-bot

**Comment by njr-11** [comment]
#10288

**Comment by njr-11** [comment]
#build

**Pull request merged**